mars/dataframe/base/apply.py

```diff
@@ -102,21 +102,29 @@ def tile(cls, op):
             for c in in_df.chunks:
                 if elementwise:
                     new_shape = c.shape
+                    new_index_value, new_columns_value = c.index_value, c.columns_value
                 else:
                     new_shape = [np.nan, np.nan]
                     new_shape[1 - axis] = c.shape[1 - axis]
+                    if axis == 0:
+                        new_index_value = parse_index(pd.RangeIndex(-1))
+                        new_columns_value = c.columns_value
+                    else:
+                        new_index_value = c.index_value
+                        new_columns_value = parse_index(pd.RangeIndex(-1))
```

Comment:
When axis == 1, due to `rechunk`, maybe the `columns_value` is just `out_df.columns_value`?